### ISSUE TITLE
Putting the bar on top blocks the system top bar 

### ISSUE THREAD

**Issue by imyxh** [micheleg/dash-to-dock#543]
Intellihide on:
![image](https://user-images.githubusercontent.com/12703838/28001410-14f47868-64e1-11e7-9d7f-b86acabe9209.png)
Intellihide off:
![image](https://user-images.githubusercontent.com/12703838/28001453-6e860a22-64e1-11e7-9d12-88d9359d7203.png)
When the bar is visible on top, it always blocks the gnome top bar. The best way to make the bar usable on top is to enable intelligent autohide and disable "dodge windows."
It would be great if the dock worked on top with no autohide (it could just sit there below the top bar) and also with intellihide, which could be implemented by having it pop down from the bottom edge of the top bar when pressure is applied to the top edge of the top bar.

**Comment by micheleg** [comment]
Yes, top positioning is not very well supported. I provide the option for people that might like it, but I've never found a satisfactory behaviour compatible with the top panel.

**Comment by nobody-special666** [comment]
it would be great if either:
1. the dock could embed itself in the top bar like the dash to panel extension 
or
2. allow the user to offset the dock position so it doesn't block the other tools on the top bar.
you already have an option to control the dock width, why not add one to control the horizontal/vertical offset of the bar along the side.
